### DISCUSSION TITLE
关于 BackTop 组件开发

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#22] [General]
设计稿：https://www.figma.com/file/3MF1lkgpW3mMzgRCngpP5C/backTop-%E7%BB%84%E4%BB%B6?node-id=0%3A1

**Comment by kiss5523** [comment]
# 最大标题

`distance-top` 距离顶部多少距离显示
`type` 显示风格 `长/圆`
`font-text` 显示文字 `长`
`icon` 自定义图标 `圆`
`duration` 回到顶部过渡时间
`position-right` 靠右位置
`position-bottom` 靠底位置


### 内置动画或后期框架加入内置动画的时候在迭代
`animation-display` 显示时的动画
`animation-hide`      隐藏时的动画

**Comment by Tyh2001** [comment]
- `type`  可以直接改为 `round` 属性，`round` 就是圆形，否则就是由内容撑开的
- `font-text` 这个我没太明白
- `icon` 自定义图标，页可以同插槽实现，不过配置项页可以使用
- `duration` 回到顶部过渡时间
- `position-right` 靠右位置 可以直接改为 `right`
- `position-bottom` 靠底位置 可以直接改为 `bottom`

**其它新增**

- `zIndex` 原生属性
- `appendToBody` 是否放到 `body` 上
